crates/ere-hosts/src/main.rs

```diff
@@ -165,39 +165,39 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     {
         run_cargo_patch_command("sp1")?;
         let sp1_zkvm = new_sp1_zkvm(resource.clone())?;
-        run_benchmark_ere("sp1", sp1_zkvm, action, &corpuses);
+        run_benchmark_ere("sp1", sp1_zkvm, action, &corpuses)?;
         ran_any = true;
     }
 
     #[cfg(feature = "zisk")]
     {
         run_cargo_patch_command("zisk")?;
-        let zisk_zkvm = new_zisk_zkvm(resource)?;
-        run_benchmark_ere("zisk", zisk_zkvm, action, &corpuses);
+        let zisk_zkvm = new_zisk_zkvm(resource.clone())?;
```

Comment:
This was a miss from a [previous PR](https://github.com/eth-act/zkevm-benchmark-workload/pull/81/files#diff-e29b59ae50fec16f88b645508dbccf764f11d9e9ee663ebca69fef48fbcb4673R175).